README.md

```diff
@@ -1,2 +1,75 @@
-# RACTF Backend
-The backend API for the RACTF platform.
+<h1 align="center">RACTF Core</h1>
+
+<div align="center">
+  🚩⛳🏁🏳️‍🌈
+</div>
+<div align="center">
+  <strong>A robust CTF platform</strong>
+</div>
+<div align="center">
+  A <strong>Really Awesome</strong> framework for hosting Cyber-Security Capture the Flag events
+</div>
+
+<div align="center">
+  <h3>
+    <a href="https://ractf.co.uk">
+      Website
+    </a>
+    <span> | </span>
+    <a href="https://docs.ractf.co.uk">
+      Docs
+    </a>
+    <span> | </span>
+    <a href="https://github.com/ractf/installer">
+      Installer
+    </a>
+    <span> | </span>
+    <a href="https://github.com/ractf/core/blob/master/CODE_OF_CONDUCT.md">
+      Code of Conduct
+    </a>
+    <span> | </span>
+    <a href="https://twitter.com/RACTF_UK">
+      Twitter
+    </a>
+    <span> | </span>
+    <a href="https://discord.gg/FfW2xXR">
+      Chat
+    </a>
+  </h3>
+</div>
+
+<div align="center">
+  <sub>The CTF framework that <strong>won't</strong> fall over half way through an event. Built with ❤︎ by
```

Comment:
This seems to implies other frameworks can stay up until half way through the event?